iohub/ndtiff.py

```diff
@@ -140,32 +140,21 @@ def get_zarr(self, position: int) -> zarr.array:
         # TODO: try casting the dask array into a zarr array
         # using `dask.array.to_zarr()`.
         # Currently this call brings the data into memory
```

Comment:
I believe now `as_array` returns a dask array without bring the data into memory, I think you can test and remove this comment

Reply:
I *think* this means that `dask.array.to_zarr()` brings data to memory if the store is a `MemoryStore` so this method cannot be consistent with other `get_zarr()` methods.

Reply:
Hopefully this won't be a problem anymore after #132 and we refactor this.